### PR TITLE
feat(profile): add route to create a profile

### PR DESCRIPTION
**Pull request by johntowell**
Closes: CU2-98

@eweise @antoinemeyer @olmstead @abuecker

### CODE REVIEW

**Comment by eweise** [comment]
👍

**Comment by abuecker** [comment]
Do we need to send the firm id?

File: api/v3/schema/profile.req.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  "properties": {
+    "email": {
+      "type": "string",
+      "format": "email"
+    },
+    "password": {
+      "type": "string"
+    }
+  }
+}
```
Comment (abuecker) [inline api/v3/schema/profile.req.json]: Need required array.

Comment (johntowell) [inline api/v3/schema/profile.req.json]: yes, I will add that

**Comment by johntowell** [comment]
@abuecker interesting. We use the origin/domain for firm in the consumer, but we might want to have the ability to pass in the firm id directly? Although I don't know of any routes that return it currently.

**Pull request merged**